src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,31 @@
-export const TodoList = () => {};
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
```

Comment:
The `TodoInfo` component should be imported as a default import. Change the import statement to `import TodoInfo from '../TodoInfo/TodoInfo';` to match the default export.